### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by iygwei** [1Panel-dev/1Panel#6468] [Q&A]
### 联系方式

iygwei@qq.com

### 1Panel 版本

v1.10.14-lts

### 问题描述

Key: 'MysqlVariablesUpdate.Type' Error:Field validation for 'Type' failed on the 'required' tag Key: 'MysqlVariablesUpdate.Database' Error:Field validation for 'Database' failed on the 'required' tag 
![image](https://github.com/user-attachments/assets/912ed2aa-860a-41c5-a868-b86d9912399b)
开启sql慢日志的时候报错

### 重现步骤

再sql卡其慢日志的时候报错的

**Comment by ssongliu** [comment]
感谢反馈，在 1.10.14-lts 版本上暂时未复现该问题，可以加我提供一下远程环境，我看看
![image](https://github.com/user-attachments/assets/f780ade6-ca83-4783-805a-e33e63d31a93)

**Reply by iygwei** [reply]
> 感谢反馈，在 1.10.14-lts 版本上暂时未复现该问题，可以加我提供一下远程环境，我看看 ![image](https://private-user-images.githubusercontent.com/73214554/366757415-f780ade6-ca83-4783-805a-e33e63d31a93.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxMjQ5MTIsIm5iZiI6MTcyNjEyNDYxMiwicGF0aCI6Ii83MzIxNDU1NC8zNjY3NTc0MTUtZjc4MGFkZTYtY2E4My00NzgzLTgwNWEtZTMzZTYzZDMxYTkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTEyVDA3MDMzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEzZTgwZTJhMDg3YTg5OTAwODZjNWY2NDgyZDY5MjRmMmEzMjdjOWViNTg4N2QyYjc0MWM3ZWJkMjQyOWE1ZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.th_Z84EzhjvJnnIA6Yt09e0lsGlFuGYKhK8u6a-DCHA)

我也给不了你看了我升级到v1.10.15-beta.5 就没了

**Reply by ssongliu** [reply]
> > 感谢反馈，在 1.10.14-lts 版本上暂时未复现该问题，可以加我提供一下远程环境，我看看 ![image](https://private-user-images.githubusercontent.com/73214554/366757415-f780ade6-ca83-4783-805a-e33e63d31a93.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxMjQ5MTIsIm5iZiI6MTcyNjEyNDYxMiwicGF0aCI6Ii83MzIxNDU1NC8zNjY3NTc0MTUtZjc4MGFkZTYtY2E4My00NzgzLTgwNWEtZTMzZTYzZDMxYTkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTEyVDA3MDMzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEzZTgwZTJhMDg3YTg5OTAwODZjNWY2NDgyZDY5MjRmMmEzMjdjOWViNTg4N2QyYjc0MWM3ZWJkMjQyOWE1ZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.th_Z84EzhjvJnnIA6Yt09e0lsGlFuGYKhK8u6a-DCHA)
> 
> 我也给不了你看了我升级到v1.10.15-beta.5 就没了

1.10.15-beta.5 没问题了吗

**Reply by iygwei** [reply]
> > > 感谢反馈，在 1.10.14-lts 版本上暂时未复现该问题，可以加我提供一下远程环境，我看看 ![image](https://private-user-images.githubusercontent.com/73214554/366757415-f780ade6-ca83-4783-805a-e33e63d31a93.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxMjQ5MTIsIm5iZiI6MTcyNjEyNDYxMiwicGF0aCI6Ii83MzIxNDU1NC8zNjY3NTc0MTUtZjc4MGFkZTYtY2E4My00NzgzLTgwNWEtZTMzZTYzZDMxYTkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTEyVDA3MDMzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEzZTgwZTJhMDg3YTg5OTAwODZjNWY2NDgyZDY5MjRmMmEzMjdjOWViNTg4N2QyYjc0MWM3ZWJkMjQyOWE1ZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.th_Z84EzhjvJnnIA6Yt09e0lsGlFuGYKhK8u6a-DCHA)
> > 
> > 
> > 我也给不了你看了我升级到v1.10.15-beta.5 就没了
> 
> 1.10.15-beta.5 没问题了吗

升级以后就没问题了

**Reply by iygwei** [reply]
> > > > 感谢反馈，在 1.10.14-lts 版本上暂时未复现该问题，可以加我提供一下远程环境，我看看 ![image](https://private-user-images.githubusercontent.com/73214554/366757415-f780ade6-ca83-4783-805a-e33e63d31a93.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxMjQ5MTIsIm5iZiI6MTcyNjEyNDYxMiwicGF0aCI6Ii83MzIxNDU1NC8zNjY3NTc0MTUtZjc4MGFkZTYtY2E4My00NzgzLTgwNWEtZTMzZTYzZDMxYTkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTEyVDA3MDMzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEzZTgwZTJhMDg3YTg5OTAwODZjNWY2NDgyZDY5MjRmMmEzMjdjOWViNTg4N2QyYjc0MWM3ZWJkMjQyOWE1ZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.th_Z84EzhjvJnnIA6Yt09e0lsGlFuGYKhK8u6a-DCHA)
> > > 
> > > 
> > > 我也给不了你看了我升级到v1.10.15-beta.5 就没了
> > 
> > 
> > 1.10.15-beta.5 没问题了吗
> 
> 升级以后就没问题了
还有台服务器有问题 但是不是同一个 一直提示的是超时 我加你了 你看看
